content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -43,6 +43,35 @@ In Odoo
   and the zip code). Go to :menuselection:`Settings --> Users & Companies --> Companies`
   to open and edit your Company record.
 
+Automatically post taxes to the correct Tax Payable account
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* For each of your companies which uses TaxCloud, you need to create a User-Defined Default 
+  so that the new taxes generated by the TaxCloud integration are created with the correct Tax Payable account:
+* In your company's Chart of Accounts, take note of the ID of the Tax Payable account you want to use.
+
+.. image:: media/taxcloud_user_defined_default_01.png
```

Comment:
Is this image necessary? 
If so, please check our guidelines: https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#images
Imo, you could remove this image and replace it with some text.

Also, your images should be in another file as mentioned in our guidelines: https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#media-files

Reply:
I have tried to clarify this in the text as you suggested, but even then I doubt it would be clear enough for the reader if they are not used to finding the ID of a record through the URL string. To be safe, I think it's worth keeping the image (though I have resized it so as to focus on the URL bar).